### PR TITLE
MEB-35: New customer state: waiting

### PR DESCRIPTION
**Pull request by gentlecat**
*Fixes [MEB-35](http://tickets.musicbrainz.org/browse/MEB-35).*

### CODE REVIEW

File: metabrainz/model/user.py [hunk 1]
```diff
@@ -11,15 +11,17 @@
...
 STATE_REJECTED = "rejected"
 
 
 class User(db.Model, UserMixin):
     """User model is used for users of MetaBrainz services like Live Data Feed.
 
-    Users are ether commercial or non-commercial (see `is_commercial`). Their access to the API is
-    determined by their `state` (active, pending, or rejected). All non-commercial users have
-    active state by default, but commercial users need to be approved by one of the admins first.
+    Users are ether commercial or non-commercial (see `is_commercial`). Their
```
Comment (mayhem) [inline metabrainz/model/user.py]: spelling: "either"

**Comment by mayhem** [comment]
Yay, can't wait for this! :+1:

**Comment by mayhem** [comment]
And please deploy when you get a chance.

File: metabrainz/model/user.py (hunk 1 above)
Comment (gentlecat) [inline metabrainz/model/user.py]: Fixed. Thanks. :+1:

**Pull request merged**